README.md

```diff
@@ -1,66 +1,66 @@
 ===========================
 
-Thank you for choosing Yii 2 - a modern PHP framework designed for professional Web development.
+Thank you for choosing Yii2 - a modern PHP framework designed for professional Web development.
+Yii2 inherits the main spirit behind Yii for being highly extensible, secure and fast.
+We strive to embrace all the best practices applied in modern Web application development.
 
-Yii 2 is a complete rewrite of its previous version Yii 1.1 which is one of the most popular PHP frameworks.
-Yii 2 inherits the main spirit behind Yii for being simple, fast and highly extensible.
-Yii 2 requires PHP 5.4 and embraces the best practices and protocols found in modern Web application development.
-
-
-[![Latest Stable Version](https://poser.pugx.org/yiisoft/yii2/v/stable.png)](https://packagist.org/packages/yiisoft/yii2)
-[![Total Downloads](https://poser.pugx.org/yiisoft/yii2/downloads.png)](https://packagist.org/packages/yiisoft/yii2)
-[![Reference Status](https://www.versioneye.com/php/yiisoft:yii2/reference_badge.svg)](https://www.versioneye.com/php/yiisoft:yii2/references)
 [![Build Status](https://img.shields.io/travis/yiisoft/yii2.svg)](http://travis-ci.org/yiisoft/yii2)
+[![Reference Status](https://www.versioneye.com/php/yiisoft:yii2/reference_badge.svg)](https://www.versioneye.com/php/yiisoft:yii2/references)
 [![Dependency Status](https://www.versioneye.com/php/yiisoft:yii2/dev-master/badge.png)](https://www.versioneye.com/php/yiisoft:yii2/dev-master)
 [![HHVM Status](https://img.shields.io/hhvm/yiisoft/yii2-dev.svg)](http://hhvm.h4cc.de/package/yiisoft/yii2-dev)
 [![Code Coverage](https://scrutinizer-ci.com/g/yiisoft/yii2/badges/coverage.png?s=31d80f1036099e9d6a3e4d7738f6b000b3c3d10e)](https://scrutinizer-ci.com/g/yiisoft/yii2/)
 [![Scrutinizer Quality Score](https://scrutinizer-ci.com/g/yiisoft/yii2/badges/quality-score.png?s=b1074a1ff6d0b214d54fa5ab7abbb90fc092471d)](https://scrutinizer-ci.com/g/yiisoft/yii2/)
 [![Code Climate](https://img.shields.io/codeclimate/github/yiisoft/yii2.svg)](https://codeclimate.com/github/yiisoft/yii2)
 
-DIRECTORY STRUCTURE
--------------------
 
-```
-build/               internally used build tools
-docs/                documentation
-framework/           core framework code
-tests/               tests of the core framework code
-```
-
-
-REQUIREMENTS
+INSTALLATION
 ------------
 
-The minimum requirement by Yii is that your Web server supports PHP 5.4.
+**Requirements**
 
+Yii2 requires `PHP 5.4`. 
 
-DOCUMENTATION
--------------
+**Composer**
 
-Yii 2.0 has a [Definitive Guide](http://www.yiiframework.com/doc-2.0/guide-index.html) and 
-a [Class Reference](http://www.yiiframework.com/doc-2.0/index.html) which cover every detail of Yii.
+With [Composer](https://getcomposer.org/) installed, you can install Yii by running the following commands under a Web-accessible folder:
```

Comment:
There's no need to run it under web accessible folder. In contrary, doing that would result in application files right under webroot which is not what we should encourage.